src/components/mobile-menu-levels-generic/index.tsx

```diff
@@ -27,14 +28,17 @@ import s from './mobile-menu-levels-generic.module.css'
 function MobileMenuLevelsGeneric() {
 	return (
 		<MobileMenuContainer className={s.mobileMenuContainer}>
-			<MobileAuthenticationControls />
-			<ul className={s.mobileMenuNavList}>
-				<SidebarNavMenuItem item={{ heading: 'Main Menu' }} />
-				{generateTopLevelSubNavItems().map((item: $TSFixMe, index: number) => (
-					// eslint-disable-next-line react/no-array-index-key
-					<SidebarNavMenuItem item={item} key={index} />
-				))}
-			</ul>
+			<div className={s.mobileMenuAuthContainer}>
+				<MobileAuthenticationControls />
+			</div>
```

Comment:
```suggestion
			<div className={s.mobileMenuAuthContainer}>
				<MobileAuthenticationControls />
			</div>
```

Reply:
Unfortunately that component doesn't support `className` in its types or props. I can add support, however! It would be a more elegant solution than the extra div for sure.

Reply:
Support added here:
9ceef62

With a follow-up commit to tweak the `margin-bottom` here, though it might even be best to remove `margin-bottom` on the component proper 🤔 
8300524